contents/blog/the-posthog-array-1-31-0.md

```diff
@@ -0,0 +1,143 @@
+
+Group Analytics is especially useful if you have a B2B product, as you will now be able to create a Company group type which tracks all unique users within a company, then create insights such as retention by company and events performed by unique companies. 
+
+Visit our [Group Analytics documentation](/docs/user-guides/group-analytics) to find out what else is possible with Group Analytics. 
+
+
+<img src="https://posthog-static-files.s3.us-east-2.amazonaws.com/Website-Assets/Array/group-analytics-list.png" alt="Example screenshot: List of groups" />
+
+
+<img src="https://posthog-static-files.s3.us-east-2.amazonaws.com/Website-Assets/Array/group-analytics-insight.png" alt="Example screenshot: Using groups in an insight" />
+
+> 🎁 Group Analytics is a premium feature and requires a PostHog Scale or Enterprise license. [Learn more](/pricing).
+
+<br />
+
+### Improved: Correlation Analysis
+
+Correlation Analysis just got better! In addition to significantly improving the UI & UX, we've also introduced a details option for advanced users looking for deeper understanding of how events and properties contribute to conversion or drop-offs. This new detail view shows a complete confusion matrix which shows true positives, true negatives, false negatives and false positives. We've also added a correlation score from `-1.0` to `1.0` to signal how strongly an event or property correlates with conversion or drop-off.
+
+<img src="https://posthog-static-files.s3.us-east-2.amazonaws.com/Website-Assets/Array/correlation-matrix.png" alt="Example screenshot: correlation matrix" />
+
+
+<br />
+
+### Polished: Experience of insights
+
+Insights and dashboards are the core of PostHog's analytics capabilities, which is why we're putting extra focus on making using them _spark joy_. This release brings various improvements to the experience:
+
+- You can now easily link to saved insights like so: https://app.posthog.com/insights/vMA1IlmP. While unwieldy query parameters were previously required, now all PostHog wants for Christmas is the ID of the insight. Merry sharing!
+- Visualization of funnels has been reworked for improved readability of results, particularly when using breakdown. This makes comparing conversion based on properties easier than ever.
+- Searching events & properties is now significantly faster. We've changed the way these properties are shown and now list them by popularity within the project. Create those insights faster!
+- The funnel query builder has been streamlined – essential settings are better exposed, while advanced options can be expanded when you need them.
+- The dashboard grid has been simplified to avoid annoying situations where your carefully crafted layout becomes misaligned on a different sceren resolution. Instead of four complicated layouts, there are now two: single-column for mobile devices, and multi-column for larger screens.
```

Comment:
```suggestion
- The dashboard grid has been simplified to avoid annoying situations where your carefully crafted layout becomes misaligned on a different screen resolutions. Instead of four complicated layouts, there are now two: single-column for mobile devices, and multi-column for larger screens.
```